code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -589,6 +591,38 @@ Congratulations! You are now trained for xenobiology research!"}
 	canSmoothWith = null
 	frame = /obj/structure/table_frame/abductor
 
+/obj/structure/table/optable/abductor
+	name = "alien operating table"
+	desc = "Used for alien medical procedures. The surface is covered in tiny spines."
+	frame = /obj/structure/table_frame/abductor
+	buildstack = /obj/item/stack/sheet/mineral/silver
+	framestack = /obj/item/stack/sheet/mineral/abductor
+	buildstackamount = 1
+	framestackamount = 1
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "bed"
+	can_buckle = 1
+	buckle_lying = 1
+
+	var/list/injected_reagents = list("corazone")
```

Comment:
make this static, since it doesn't change (it doesn't change dynamically, and it may in the future but until then it should be static)